nats-spring-cloud-stream-binder/src/test/java/io/nats/cloud/stream/binder/BinderTests.java

```diff
@@ -41,11 +35,16 @@
 import org.springframework.cloud.stream.provisioning.ProducerDestination;
 import org.springframework.integration.channel.DirectChannel;
 import org.springframework.messaging.MessageHandler;
+import org.springframework.messaging.MessageHeaders;
 import org.springframework.messaging.support.GenericMessage;
 
 import io.nats.client.Connection;
 import io.nats.client.Message;
 import io.nats.client.Subscription;
+import org.springframework.messaging.support.MessageBuilder;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.*;
```

Comment:
The advantage of explicitly listing all imports is that I can tell at a glance which class you meant to use, which simply makes reading the code that much easier. If you're just doing a quick one-off thing, there's nothing explicitly wrong, but future maintainers will thank you for your clarity otherwise.

Reply:
Sure, although this is a highly subjective matter and I haven't seen the repo define its approach. Nevertheless - changed it to explicit imports.
Btw.
Something like a [.editorconfig](https://editorconfig.org/) file would allow everyone to hold to the same standard.

Reply:
You are right. Thx for the hint with the editorconfig